packages/spruce-next-helpers/package.json

```diff
@@ -72,6 +72,7 @@
 	"dependencies": {
 		"@babel/polyfill": "^7.0.0",
 		"@babel/register": "^7.0.0",
+		"@sprucelabs/react-heartwood-components": "7.8.1-pre-1be2f60.0",
```

Comment:
@kengoldfarb I had to specify a prerelease to get this to work for my skill; unsure if lerna handles upgrades to this JSON automagically (I think you said it did?)